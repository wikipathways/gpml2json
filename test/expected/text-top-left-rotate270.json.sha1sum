bdf1cd63cd0009e38086cdddb0da7617abaa1259 ?-
