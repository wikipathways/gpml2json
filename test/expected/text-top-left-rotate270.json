{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e933b"],"drawAs":"rect","gpmlElementName":"Pathway","height":212.1276595744681,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":554.2553191489361,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20171102","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/1e5329c4bfd8e937483a59fcf929accccf136bca5e91169268ad81c4a639935d"},"entityMap":{"e933b":{"id":"e933b","textContent":"Label for\nthe rendering of\ntext","textAlign":"left","x":-3.0851063829786654,"y":77.76595744680857,"backgroundColor":"#ffffcc","fontSize":10,"height":100.95744680851064,"rotation":270,"drawAs":"rect","borderRadius":15,"width":156.27659574468078,"zIndex":16384,"color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"verticalAlign":"top","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"textRotation":-270,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"}}}
